scripts/codestyle.bash

```diff
@@ -28,10 +34,13 @@ ci() {
   isort --check setup.py "$BASE_PATH_DIR"/src/"$SRC_DIRECTORY_NAME" "$BASE_PATH_DIR"/tests
   echo "black"
   black --check "$BASE_PATH_DIR"/src/"$SRC_DIRECTORY_NAME" "$BASE_PATH_DIR"/tests
-  echo "pylint"
-  pylint --rcfile=.pylintrc "$BASE_PATH_DIR"/src/"$SRC_DIRECTORY_NAME" "$BASE_PATH_DIR"/tests
-  echo "mypy"
-  mypy --config-file mypy.ini --ignore-missing-imports "$BASE_PATH_DIR"/src/"$SRC_DIRECTORY_NAME" "$BASE_PATH_DIR"/tests
+  echo "pylint ..."
+  pylint --rcfile="$PYLINT_CONFIG" "$BASE_PATH_DIR"/src/"$SRC_DIRECTORY_NAME" "$BASE_PATH_DIR"/tests
+  echo "mypy ..."
+  # installing all missing stub packages (e.g. types-PyYAML, types-aiofiles, etc)
+  python3 -m pip install types-aiofiles types-PyYAML types-ujson
+  # runs mypy
+  mypy --config-file "$MYPY_CONFIG" --ignore-missing-imports "$BASE_PATH_DIR"/src/"$SRC_DIRECTORY_NAME" "$BASE_PATH_DIR"/tests
```

Comment:
still don't get why we use 2 different versions of mypy... we should unify

Reply:
Will make sure they are the same in a separate PR